src/soep_preparation/clean_variables/pl.py

```diff
@@ -31,6 +32,47 @@ def _private_rente_beitrag_jahr(
     return out.where(eingezahlt.astype("bool[pyarrow]"), 0)
 
 
+def _private_rente_beitrag_monatlich(data: pd.DataFrame) -> pd.Series:
+    private_rente_beitrag_m = pd.Series(
+        data["private_rente_beitrag_m_2013"].where(
+            data["survey_year"] != 2018,  # noqa: PLR2004
+            data["private_rente_beitrag_m_2018"],
+        ),
+        name="private_rente_beitrag_m",
+    )
+
+    data = pd.concat([data["p_id"], private_rente_beitrag_m], axis=1)
+    out = data.groupby("p_id")["private_rente_beitrag_m"].ffill()
+    return apply_smallest_float_dtype(out)
+
+
+def _frailty(data: pd.DataFrame) -> pd.Series:
+    med_vars = [
+        "med_schwierigkeit_treppen_pl",
+        "med_schwierigkeit_taten_pl",
+        "med_schlaf_pl",
+        "med_diabetes_pl",
+        "med_asthma_pl",
+        "med_herzkrankheit_pl",
+        "med_krebs_pl",
+        "med_schlaganfall_pl",
+        "med_migräne_pl",
+        "med_bluthochdruck_pl",
+        "med_depressiv_pl",
+        "med_demenz_pl",
+        "med_gelenk_pl",
+        "med_rücken_pl",
+        "med_sonst_pl",
+        "med_raucher_pl",
+    ]
+
+    med_var_data = pd.concat(
+        [data[med_vars], data["med_subjective_status_dummy_pl"]],
+        axis=1,
+    )
+    return apply_smallest_float_dtype(med_var_data.mean(axis=1))
```

Comment:
```suggestion
    return apply_smallest_float_dtype(
        data[
            "med_schwierigkeit_treppen_pl",
            "med_schwierigkeit_taten_pl",
            "med_schlaf_pl",
            "med_diabetes_pl",
            "med_asthma_pl",
            "med_herzkrankheit_pl",
            "med_krebs_pl",
            "med_schlaganfall_pl",
            "med_migräne_pl",
            "med_bluthochdruck_pl",
            "med_depressiv_pl",
            "med_demenz_pl",
            "med_gelenk_pl",
            "med_rücken_pl",
            "med_sonst_pl",
            "med_raucher_pl",
            "med_subjective_status_dummy_pl"
        ].mean(axis=1),    )
```

But see my second thought above.